### PR TITLE
ft: #184361206 Setup docker for backend application

### PR DESCRIPTION
**Pull request by musllim**
This changes setup the following services on local development:

1. express server
2. postgress db.

# Description

there is nothing worry on how heroku will run the app. this is only valid when we run locally, heroku will run the application in normal way

Fixes # (issue)

# Type of change

Please check the options that are relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature or Enhancement (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have created a loom video for the new feature or enhancement 
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by emilenfc — APPROVED** [review]
That's nice, LGTM

**Review by SergeInGithub — DISMISSED** [review]
LGTM

**Review by shyakadavis — APPROVED** [review]
LGTM 🔬 🚀

**Review by shyakadavis — APPROVED** [review]
LGTM 🌤️

**Review by SergeInGithub — DISMISSED** [review]
LGTM

**Review by shyakadavis — DISMISSED** [review]
LGTM 🚀

**Review by emilenfc — APPROVED** [review]

**Review by shyakadavis — DISMISSED** [review]
LGTM 🏙️

**Review by SergeInGithub — DISMISSED** [review]
LGTM

File: Dockerfile [hunk 1]
```diff
@@ -1,18 +1,13 @@
-FROM node:19.5
+FROM node:19.5.0-alpine
```
Comment (mucyoalexaxel) [inline Dockerfile:1]: use the LTS version of node or the same version you have in your docker-compose file. @musllim

File: Dockerfile [hunk 2]
```diff
@@ -1,18 +1,13 @@
...
-ADD https://github.com/ufoscout/docker-compose-wait/releases/download/2.8.0/wait /wait
+EXPOSE 8080
 
-RUN chmod +x /wait
-
-EXPOSE 9090
-
-CMD /wait && pnpm start
\ No newline at end of file
+CMD yarn migrate:dev && yarn seed:dev && yarn dev
```
Comment (mucyoalexaxel) [inline Dockerfile:13]:
```suggestion
CMD yarn migrate:dev && yarn seed:dev && yarn dev

```
always have a new line at the end of your code

**Review by mucyoalexaxel — CHANGES_REQUESTED** [review]
Great work here @musllim . Kindly address the suggested change and everything should be good

File: Dockerfile [hunk 3]
```diff
@@ -1,18 +1,13 @@
-FROM node:19.5
+FROM node
```
Comment (mucyoalexaxel) [inline Dockerfile:1]:
```suggestion
FROM node:14
``````

**Review by mucyoalexaxel — CHANGES_REQUESTED** [review]

**Review by mucyoalexaxel — APPROVED** [review]
LGTM

**Pull request merged**